acceptance/openstack/cbr/v3/tags_test.go

```diff
@@ -40,52 +49,31 @@ func TestTags(t *testing.T) {
 		th.AssertNoErr(t, vaults.Delete(client, vault.ID).ExtractErr())
 	}()
 
-	projectTags, err := tags.ShowVaultProjectTag(client).Extract()
+	projectTags, err := cbrtags.ShowVaultProjectTag(client).Extract()
```

Comment:
better to call module tags not cbrtags because it is already under cbr package

Reply:
Then I'll add an alias.